### ISSUE TITLE
Create initial alpha release for vsl.plot

### ISSUE THREAD

**Issue by ulises-jeremias** [vlang/vsl#52]
@lucasrdrgs is already doing this work and sent some really good demos on discord. This ticket is just to track this work for v0.2. Having this work and all existing modules with tests passing will be a huge part of the existing roadmap for the next minor release.

- @lucasrdrgs to continue implementing the library
- @ulises-jeremias to define the deps installation for the new module `vsl.plot`. @lucasrdrgs defined an script `create-venv.sh` so @ulises-jeremias will define the best way to handle that script in an automated way using existing VSL automation.
- @ulises-jeremias to update deps on official docker containers

Once this ticket is finished, it is needed to create new github issues for future work to make it possible that all structure on VSL has the method `.plot()` and we have a `Plottable` interface, so it does not matters if the user knows about data plotting or not, they will have the possibility to use a predefined plotting for all structs (graph, Kmeans, KNN, LinReg, etc)
